art/metrics/metrics.py

```diff
@@ -77,13 +85,15 @@ def empirical_robustness(classifier, x, attack_name, attack_params=None):
     :type classifier: :class:`.Classifier`
     :param x: Data sample of shape that can be fed into `classifier`
     :type x: `np.ndarray`
-    :param attack_name: A string specifying the attack to be used. Currently supported attacks are {`fgsm'}
-                        (Fast Gradient Sign Method)
+    :param attack_name: A string specifying the attack to be used. Currently supported attacks are {`fgsm', `hsj`}
+                        (Fast Gradient Sign Method, Hop Skip Jump)
     :type attack_name: `str`
     :param attack_params: A dictionary with attack-specific parameters. If the attack has a norm attribute, then it will
                           be used as the norm for calculating the robustness; otherwise the standard Euclidean distance
                           is used (norm=2).
     :type attack_params: `dict`
+    :param round: Round the features if they are one-hot encoded
+    :type round: 'bool'
```

Comment:
Is the documentation for `round` still required? If not, please remove it.

Reply:
Removed